pyaedt/application/Analysis3D.py

```diff
@@ -1035,18 +1035,33 @@ def flatten_3d_components(self, component_name=None, purge_history=True, passwor
             self.mesh._refresh_mesh_operations()
         return True
 
-    def identify_touching_conductors(self):
-        # type: () -> dict
+    def identify_touching_conductors(self, object_name=None):
+        # type: (str) -> dict
         """Identify all the touching components and group in a dictionary. It requires `pyvista`.
 
+        Parameters
+        ----------
+        object_name : str
+            Starting object to check for touching elements. Optional.
```

Comment:
```suggestion
        object_name : str, optional
            Starting object to check for touching elements. The default is ``None``.
```
Can we describe the default behavior using  this format: The default is ``None``, in which case ....